lapis2-docs/src/components/ConfigGenerator/AdditionalInformationWizard.tsx

```diff
@@ -0,0 +1,81 @@
+import { type FC, useContext, useMemo, useEffect } from 'react';
+import { ConfigContext, type MetadataType } from './configContext.tsx';
+import { sentenceCase } from 'change-case';
+
+export function AdditionalInformationWizard() {
+    const { configType, config, modifyConfigField, removeConfigField } = useContext(ConfigContext);
+
+    useEffect(() => {
+        const anyStringMetadata = config.metadata.find((metadata) => metadata.type === 'string');
+        if (anyStringMetadata !== undefined) {
+            modifyConfigField('primaryKey', anyStringMetadata.name);
+        }
+
+        const anyDateMetadata = config.metadata.find((metadata) => metadata.type === 'date');
+        if (anyDateMetadata !== undefined) {
+            modifyConfigField('dateToSortBy', anyDateMetadata.name);
+        }
+
+        const anyPangoMetadata = config.metadata.find((metadata) => metadata.type === 'pango_lineage');
+        if (anyPangoMetadata !== undefined) {
+            modifyConfigField('partitionBy', anyPangoMetadata.name);
+        }
+    }, []);
+
+    return (
+        <div className='mb-4'>
+            <h1 className='text-xl font-bold mb-4'>Additional Information</h1>
+            <table className='table'>
+                <tbody>
+                    <MetadataDropDown filterByType={'string'} name={'primaryKey'} />
+                    <MetadataDropDown filterByType={'date'} name={'dateToSortBy'} />
+                    <MetadataDropDown filterByType={'pango_lineage'} name={'partitionBy'} />
+                </tbody>
+            </table>
+        </div>
+    );
+}
+
+type MetadataDropDownProps = {
+    name: 'primaryKey' | 'dateToSortBy' | 'partitionBy';
+    filterByType?: MetadataType;
+};
+const MetadataDropDown: FC<MetadataDropDownProps> = ({ name, filterByType }) => {
+    const { config, modifyConfigField } = useContext(ConfigContext);
+
+    const options = useMemo(
+        () =>
+            config.metadata!.filter((metadata) => {
+                if (filterByType === undefined) {
+                    return true;
+                }
+                return metadata.type === filterByType;
+            }),
+        [config.metadata, filterByType],
+    );
+
+    const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
Let's also import this?
```suggestion
    const handleChange = (event: ChangeEvent<HTMLSelectElement>) => {
```